api/cds.proto

```diff
@@ -230,4 +231,44 @@ message Cluster {
   // This overrides any bind_config specified in the bootstrap proto.
   // If the addres and port are empty, no bind will be performed.
   BindConfig upstream_bind_config = 21;
+
+  // Optionally divide the endpoints in this cluster into subsets defined by
+  // endpoint metadata and selected by route and weighted cluster metadata.
+  message LbSubsetConfig {
+    // The behavior used when no endpoint subset matches the selected route's
+    // metadata. The options are no_fallback, any_endpoint, or default_subset.
```

Comment:
Nit: vague preference to capitalize these comments to match the enum. When we auto-generate docs, it will look more consistent.

Reply:
Heh. I was following the pattern in DnsLookupFamily. Want me to fix those up too?

Reply:
Yeah. A bunch of docs were copy+paste from the v1 JSON, which may have not been case sensitive. In v2, folks are going to have to use all caps for JSON/YAML/proto.